### PR TITLE
Revision of brain atlas

### PR DESCRIPTION
**Pull request by lzehl**
PR according to the discussions in #29 
Please verify and double check.

### CODE REVIEW

File: v0.1/core/brainAtlasVersion.schema.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    "@type",
+    "@id",
+    "alias",
+    "annotationSet",
+    "fullName",
+    "releaseDate",
+    "terminology",
+    "versionIdentifier",
+    "versionInnovation" 
+  ],
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:17]: According to issue #29, `coordinateSpace` is also required.

File: v0.1/core/brainAtlasVersion.schema.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/core/brainAtlas.schema.json",
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:2]: Wrong schema name. Change brainAtlas.schema.json --> brainAtlasVersion.schema.json.

File: v0.1/core/brainAtlasVersion.schema.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    "releaseDate",
+    "terminology",
+    "versionIdentifier",
+    "versionInnovation" 
+  ],
+  "properties": {
+    "@type": {
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://schema.ebrains.eu/sands/BrainAtlas"
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:22]: Wrong name. Change BrainAtlas --> BrainAtlasVersion.

File: v0.1/core/brainAtlasVersion.schema.json [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier (id).",
+      "pattern": "sands/core/brainatlas/([A-Za-z0-9_-]+).json"
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:27]: Wrong name. Change brainatlas --> brainatlasversion

File: v0.1/core/brainAtlasVersion.schema.json [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    },
+    "alias": {
+      "type": "string",
+      "description": "User-defined identifier, often a short name, of this brain atlas."
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:31]: "[...] of this brain atlas VERSION."

File: v0.1/core/brainAtlasVersion.schema.json [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    },
+    "alternativeVersionOf": {
+      "description": "Link to an alternative brain atlas version.",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/BRAIN_ATLAS_VERSION",
+    },
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:36]: Define `"type": "array"`. According to issue #29, count is 0 - n.

File: v0.1/core/brainAtlasVersion.schema.json [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    "annotationSet": {
+      "type": "array",
+      "description": "List of annotations that define this brain atlas version",
+      "items": {
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANNOTATION"
+      }
+    },
+    "coordinateSpace": {
+      "description": "Link to coordinate space this brain atlas version is represented in"
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:45]: comma missing

File: v0.1/core/brainAtlasVersion.schema.json [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/COORDINATE_SPACE",
+    },
+    "fullName": {
+      "type": "string",
+      "description": "A descriptive name of this brain atlas."
+    },
+    "newVersionOf": {
+      "description": "Link to the previous brain atlas version.",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/BRAIN_ATLAS_VERSION",
+    },
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:55]: Define as array. Count 0 -n.

File: v0.1/core/brainAtlasVersion.schema.json [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+    "@id",
+    "alias",
+    "annotationSet",
+    "fullName",
+    "releaseDate",
+    "terminology",
+    "versionIdentifier",
+    "versionInnovation" 
+  ],
+  "properties": {
```
Comment (UlrikeS91) [inline v0.1/core/brainAtlasVersion.schema.json:18]: Again the general comment about the type. To be consistent, we should always define the type for a property.

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
The wiki needs to be updated accordingly, of course.

File: v0.1/core/brainAtlasVersion.schema.json (hunk 1 above)
Comment (lzehl) [reply v0.1/core/brainAtlasVersion.schema.json:17]: yes. I forgot that one. I'll add it.

File: v0.1/core/brainAtlasVersion.schema.json (hunk 2 above)
Comment (lzehl) [reply v0.1/core/brainAtlasVersion.schema.json:2]: yes. forgot that one

File: v0.1/core/brainAtlasVersion.schema.json (hunk 3 above)
Comment (lzehl) [reply v0.1/core/brainAtlasVersion.schema.json:22]: see above

File: v0.1/core/brainAtlasVersion.schema.json (hunk 4 above)
Comment (lzehl) [reply v0.1/core/brainAtlasVersion.schema.json:27]: see above

File: v0.1/core/brainAtlasVersion.schema.json (hunk 5 above)
Comment (lzehl) [reply v0.1/core/brainAtlasVersion.schema.json:31]: yes

File: v0.1/core/brainAtlasVersion.schema.json (hunk 8 above)
Comment (lzehl) [reply v0.1/core/brainAtlasVersion.schema.json:55]: No. This one is sequential and can only refer to 1. Needs to be corrected in the documentation.

File: v0.1/core/brainAtlasVersion.schema.json (hunk 9 above)
Comment (lzehl) [reply v0.1/core/brainAtlasVersion.schema.json:18]: cf my comment in the other PRs. Let's figure out what is best practice here and then change it across the schemas in a PR.
Sorry that I started this :see_no_evil:

**Comment by lzehl** [comment]
I'll update the Wiki after the PR is merged. Thanks for the reminder :slightly_smiling_face:

File: v0.1/core/brainAtlasVersion.schema.json (hunk 8 above)
Comment (UlrikeS91) [reply v0.1/core/brainAtlasVersion.schema.json:59]: That seems also more logical to me, but I cross-checked against what was stated in the issue :) Then it's correct int he code, and wiki needs to be updated accordingly.

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**